### PR TITLE
fix flake and use operatorhub to install EDB

### LINKED ISSUE #5875: failing periodic test
All periodic tests are failing for more than a week :-/ 

- tests on 4.8 last success June 13th  
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/origin-ci-test/logs/periodic-ci-redhat-developer-odo-main-v4.8-integration-e2e-periodic
- tests on 4.9 last success June 14th
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/origin-ci-test/logs/periodic-ci-redhat-developer-odo-main-v4.9-integration-e2e-periodic
- tests on 4.10 last success June 15th
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/origin-ci-test/logs/periodic-ci-redhat-developer-odo-main-v4.10-integration-e2e-periodic
- tests on 4.11 last success June 14th
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/origin-ci-test/logs/periodic-ci-redhat-developer-odo-main-v4.11-integration-e2e-periodic
- tests on ??? last success June 15th
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/origin-ci-test/logs/periodic-ci-redhat-developer-odo-v2-integration-e2e-periodic

What is the last tests? The name is not saying much about what it is testing. 

**Why do we have periodic tests if no one is paying attention to them?**

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this:**

/kind tests

**What does this PR do / why we need it:**

**Which issue(s) this PR fixes:**

Fixes #5875 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: tests/integration/interactive_add_binding_test.go [hunk 1]
```diff
@@ -224,12 +224,12 @@ var _ = Describe("odo add binding interactive command tests", func() {
 			var nsWithNoService string
 
 			BeforeEach(func() {
+				nsWithNoService = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
```
Comment (valaparthvi) [inline tests/integration/interactive_add_binding_test.go:227]: Why this new variable? The operators we're using will be installed in all the namespaces.

Comment (rm3l) [reply tests/integration/interactive_add_binding_test.go:227]: This was added as part of #5936 (to specifically test service binding from a namespace with no services). From the diff, I can see that @anandrkskd has just moved the line to be called earlier, but I am not sure why this line was moved..

File: tests/integration/interactive_add_binding_test.go [hunk 2]
```diff
@@ -224,12 +224,12 @@ var _ = Describe("odo add binding interactive command tests", func() {
...
 				otherNS = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
 				addBindableKindInOtherNs := commonVar.CliRunner.Run("-n", otherNS, "apply", "-f",
 					helper.GetExamplePath("manifests", "bindablekind-instance.yaml"))
 				Expect(addBindableKindInOtherNs.ExitCode()).To(BeEquivalentTo(0))
-
-				nsWithNoService = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
+				commonVar.CliRunner.HasNamespaceProject(nsWithNoService)
```
Comment (rm3l) [inline tests/integration/interactive_add_binding_test.go:232]: `HasNamespaceProject` returns a Boolean. It might make sense to check the return value.

Comment (anandrkskd) [reply tests/integration/interactive_add_binding_test.go]: updated

**Review by rm3l — APPROVED** [review]

**Pull request merged**